app/views/snacks/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= @snack.name %></h1>
+
+<p><%= @snack.name %> info</p>
+<p>Price: <%= @snack.price %></p>
+
+
+<p>Locations</p>
+
+<%# something to find which machines have these snacks %>
```

Comment:
Ya, sounds like you know what you needed to do based off of your PR comment. Create a many to many relationship between machines and snacks using a joins table, and then you can list out all the machines that the snack is in.